tools/install.sh

```diff
@@ -83,8 +83,7 @@ echo ""
 cat <<QUESTIONS
 This install script will ask you to provide your project name. 
 If unsure, we suggest using "LORIS"
-(This will be used to define Imaging data filepaths automatically in a config.xml file for LORIS, and (optionally) for the apache configuration file.) 
-
+(This will be used IFF you want this script to configure your apache)
```

Comment:
I wouldn't expect most users running a script to know the meaning of the word "iff".

Reply:
Hi @driusan - that line was removed yesterday. Could you Update and re-review?  
(Or I can dismiss for you if you like)